src/reference/constellation/index.pug

```diff
@@ -0,0 +1,160 @@
+    ul
+        li #[strong cookie]: may contain the Beam session cookie. Its presence will cause Constellation to attempt to authenticate the user.
+        li #[strong authorization]: may contain and OAuth Bearer token to authenticate with for 3rd party apps, rather than using a cookie. This will indicate to Constellation that the user is a bot.
+        li
+            p.
+                #[strong x-sec-websocket-protocol]: if set to cnstl-gzip, Constellation may choose to send websocket frames down as binary, gzipped JSON rather than plain text. Passing this is generally handled by websocket clients themselves, and are usually configured by specifying a preferred subprotocol.
+
+            p.
+                The client may detect gzipped frames by the fact that they are binary messages and begin with the magic bytes 0x1f and 0x8b as the first and second payload byte, respectively.
+        li #[strong x-is-bot]: this must be set to true if the client is an automated bot rather than a human user and you are not using an authorization header. Failure to set this may cause the account to be banned.
+    h2#packets Packets
+    p.
+        There are three packet types: request, reply, and event. These are sent over the websocket as JSON messages. Messages to the client may be gzipped if x-supports-gzip was passed in the handshake, and messages sent to Constellation are always allowed to be gzipped.
+    h3#packets_method Method
+    p.
+        Method packets are sent from the client in a way very similar to JSON-RPC. This is the only packet the client may send to the server. A packet may look like the following:
+    p.
+        {"type": "method", "method": "divide", "params": {"numerator": 16, "denominator": 4}, "id": 123}
+    ul
+        li #[strong type] is always set to "method"
+        li #[strong method] should be the name of the method to call
+        li #[strong params] should be an object, not an array, of named arguments to pass into the method.
+        li #[strong id] may be any uint32. It’s included in the reply packet and used to correlate replies from the socket.
+    h3#packets_reply Reply
+    p.
+        Reply packets are sent in response to method packets. Replies are always sent in response to methods unless the socket closes before they may be sent. A some reply packets may look like the following:
+    +highlightStr('json','{"type": "reply", "result": 4, "error": null, "id": 123}')
+    +highlightStr('json','{"type": "reply", "result": null, "error": {"code": 1000, "message": "Cannot divide by zero"}, "id": 124}')
+    ul
+        li #[strong type] is always set to "reply"
+        li #[strong id] is set to the id passed in the corresponding method packet
+        li #[strong result] is the unstructured result of the method, or null
```

Comment:
You can use `#[code text]` for these. The emphasis from the doc was since the body text was already monospaced